charts/atlantis/templates/statefulset.yaml

```diff
@@ -181,7 +181,11 @@ spec:
           - name: ATLANTIS_DATA_DIR
             value: /atlantis-data
           - name: ATLANTIS_REPO_WHITELIST
+          {{- if .Values.terraformCloud.enable }}
+            value: {{ toYaml .Values.terraformCloud.orgWhitelist }}
+          {{- else }}
             value: {{ toYaml .Values.orgWhitelist }}
+          {{- end }}
           - name: ATLANTIS_PORT
             value: "4141"
```

Comment:
My knowledge is k8s is limited and old, so feel free to correct me: If we are deploying services with same port in same namespace in k8s, will that end-up with port conflict?

Reply:
This shouldn't make a difference between it should be how the `Service` will be mapped.

Reply:
This should only be specific to the container's port

Reply:
> This shouldn't make a difference between it should be how the `Service` will be mapped.

Like `svc` endpoint mapping to,
```bash
<public_ip>:4141 ----> https://atlantis-terraform-cloud:4141
<public_ip>:4141 ----> https://atlantis:4141
```
I have lot to catch-up on K8s, my last version of k8s I have dealt with were `1.10.x -> 1.12.x`

Reply:
IIRC, this should be fine. I'm not sure how the Ingress is setup in the cluster but you could have a definition that points specifically to that FQDN that service discovery has set. I think we would definitely need to look into this if we allow the public URL to be open. I've shared this PR with the orchestration team, so I'm hoping they can also speak on this.

Reply:
The ports shouldn't cause any conflict. The service is currently configured to use loadbalancerSourceRanges to allow traffic from github's webhook IP addresses https://github.com/procore/atlantis-helm-chart/blob/c1b5d1113725b202611c3cd6b644ed1bdc6b0b92/charts/atlantis/values.yaml#L169
We did not want to allow blanket public ingress without restricting the IP address range